src/test/regress/expected/multi_insert_select.out

```diff
@@ -1479,10 +1479,30 @@ COPY raw_events_first (user_id, value_1) FROM STDIN DELIMITER ',';
 ERROR:  distributed copy operations must not appear in transaction blocks containing other distributed modifications
 CONTEXT:  COPY raw_events_first, line 1: "103,103"
 ROLLBACK;
--- Views does not work
+-- selecting from views works
 CREATE VIEW test_view AS SELECT * FROM raw_events_first;
+INSERT INTO raw_events_first (user_id, time, value_1, value_2, value_3, value_4) VALUES
+                         (16, now(), 60, 600, 6000.1, 60000);
+SELECT count(*) FROM raw_events_second;
+ count 
+-------
+    11
+(1 row)
+
 INSERT INTO raw_events_second SELECT * FROM test_view;
-ERROR:  cannot plan queries that include both regular and partitioned relations
+INSERT INTO raw_events_first (user_id, time, value_1, value_2, value_3, value_4) VALUES
+                         (17, now(), 60, 600, 6000.1, 60000);
+INSERT INTO raw_events_second SELECT * FROM test_view WHERE user_id = 17 GROUP BY 1,2,3,4,5,6;
+SELECT count(*) FROM raw_events_second;
+ count 
+-------
+    13
+(1 row)
+
+-- inserting into views does not
+INSERT INTO test_view SELECT * FROM raw_events_second;
+ERROR:  cannot perform distributed planning for the given modification
+DETAIL:  Cannot modify views
```

Comment:
I think that message would be better if it were "cannot insert into view over distributed table" without any DETAIL.